modules/nw-ovn-kubernetes-observability.adoc

```diff
@@ -0,0 +1,172 @@
+----
+
+. Obtain a list of OVN-Kubernetes pods and locate the pod that shares the same `NODE` as the pods from the previous step:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-ovn-kubernetes -o wide
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME        
+...                             READY   STATUS    RESTARTS      AGE   IP           NODE                                       NOMINATED NODE   
+ovnkube-node-jzn5b              8/8     Running   1 (34m ago)   37m   10.0.128.2   ci-ln-1gqp7b2-72292-bb9dv-worker-a-gtmpc   <none>          
+...
+----
+
+. Use `oc exec` to open a bash shell inside of the `ovnkube-node` pod by entering the following command:
++
+[source,terminal]
+----
+$ oc exec -it <pod_name> -n openshift-ovn-kubernetes -- bash
+----
+
+. While inside of the `ovnkube-node` pod, you can run the `ovnkube-observ -add-ovs-collector` script to show network events using the OVS collector. For example:
++
+[source,terminal]
+----
+# /usr/bin/ovnkube-observ -add-ovs-collector
+----
++
+.Example output
++
+[source,terminal]
+----
+...
+2024/12/02 19:41:41.327584 OVN-K message: Allowed by default allow from local node policy, direction ingress
+2024/12/02 19:41:41.327593 src=10.131.0.2, dst=10.131.0.6
+
+2024/12/02 19:41:41.327692 OVN-K message: Allowed by default allow from local node policy, direction ingress
+2024/12/02 19:41:41.327715 src=10.131.0.6, dst=10.131.0.2
+...
+----
+
+. Filter the content by type, for example, source pods, by using the `-filter-src-ip` flag with your pod's IP address. For example:
++
+[source,terminal]
+----
+# /usr/bin/ovnkube-observ -add-ovs-collector -filter-src-ip <pod_ip_address>
+----
++
+.Example output
++
+[source,terminal]
+----
+...
+Found group packets, id 14
+2024/12/10 16:27:12.456473 OVN-K message: Allowed by admin network policy allow-egress-group1, direction Egress
+2024/12/10 16:27:12.456570 src=10.131.0.22, dst=10.131.0.23
+
+2024/12/10 16:27:14.484421 OVN-K message: Allowed by admin network policy allow-egress-group1, direction Egress
+2024/12/10 16:27:14.484428 src=10.131.0.22, dst=10.131.0.23
+
+2024/12/10 16:27:12.457222 OVN-K message: Allowed by network policy test:allow-ingress-from-specific-pod, direction Ingress
+2024/12/10 16:27:12.457228 src=10.131.0.22, dst=10.131.0.23
+
+2024/12/10 16:27:12.457288 OVN-K message: Allowed by network policy test:allow-ingress-from-specific-pod, direction Ingress
+2024/12/10 16:27:12.457299 src=10.131.0.22, dst=10.131.0.23
+...
+----
++
+For a full list of flags that can be passed in with `/usr/bin/ovnkube-observ`, see "OVN-Kubernetes network traffic with OVS sampling flags".
+
+[id="observability-ovs-sampling-flags_{context}"]
+== OVN-Kubernetes network traffic with OVS sampling flags
+
+The following flags are available to view OVN-Kubernetes network traffic by using the CLI. Append these flags to `ovnkube-observ` in your terminal.
+
```

Comment:
In my own work, I have included a command syntax example before the table. 

```
.Command syntax
[source,terminal]
----
$ ovnkube-observ <flag>
----
```
I know you have it in the last sentence, but sometimes I find it makes the relationship between the command and the flags a bit more scannable.

just a nit.